frontend/src/services/jobStatsServices.js

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios'
+
```

Comment:
This is a much better patter to handle the services. I'll make sure to start using this and let's try to migrate whenever we can any endpoint direclty using URL's to this wrapper functions.

Reply:
Thanks